client-app/ui-kit/components/molecules/input/vc-input.vue

```diff
@@ -58,7 +58,7 @@
           tabindex="-1"
           type="button"
           class="vc-input__password-icon"
-          @click="togglePasswordVisibility"
+          @click="isPasswordVisible = !isPasswordVisible"
```

Comment:
what if it is not a password?

Reply:
https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/ccb9648224fe6c82405704873fef51b447c28a17/client-app/ui-kit/components/molecules/input/vc-input.vue#L55

Reply:
@click="togglePasswordType"
@click="toggleSymbolsVisibility"